source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+
+## One-off payments using Direct Debit
+
+To set up a Direct Debit mandate to take a single payment:
+
+* create an API key linked to your Direct Debit service
+
+* follow the instructions in the [‘Payment Flow’ section](/payment_flow)
+
+You cannot take further payments from users using this method without creating
+a new payment. This requires users to re-enter their payment details.
+
+## Variable payments using Direct Debit
+
+A variable agreement allows you to collect payments of any amount at any time.
+For example, you may want to collect:
+
+* money each time a service is provided
+
+* payments where the amounts vary according to service usage
+
+* regular payments where you do not know in advance how much each payment will be
+
+You could also use the variable functionality to simulate a recurring or
+scheduled Direct Debit payment, by sending an API call on a set schedule.
+Users would be emailed each time a payment is collected against their mandate.
+
+You are responsible for managing variable payments using the GOV.UK Pay API.
+
+### Set up a variable Direct Debit payment
+
+1. Create an agreement using `POST /v1/agreements/`,
+specifying:
+
+* a `return_url` where GOV.UK Pay will direct the user when they have finished
+setting up the agreement
+
+* the `agreement_type` (`on demand`)
+
+* a `reference` (optional) - your reference for this agreement
+
+The response contains a `next_url`.
+
+2. Direct the user to the `next_url`.
+
+3. The user provides their payment details. For example, account number
+and sort code for Direct Debit. GOV.UK Pay makes an API call to our Direct
+Debit supplier to verify account details.
+
+4. GOV.UK Pay presents the user with a confirmation screen which presents
+their account details. The user grants their consent by selecting
+__Confirm__.  GOV.UK Pay sends an email to the user to confirm that their
+mandate has been set up. If the mandate subsequently fails, the GOV.UK Pay team
+will send an email to update the user that their mandate could not be set up.
+
+5. GOV.UK Pay securely passes the user's details on to the Direct Debit
+provider, who submit a request to BACS to set up a mandate.
+
+6. GOV.UK Pay returns the user to your service using the `return_url` from
+Step 1.
+
+7. Call `GET /v1/agreements/<AGREEMENT-ID>` to check the status of the
+agreement (in order to display an appropriate screen to the user). For a
+Direct Debit payment, this will initially be in a ‘pending’ state until the
+bank confirms the mandate. The confirmation process typically takes 3-4 days.
+
+8. When GOV.UK Pay receives confirmation that the mandate is
+confirmed, the GOV.UK Pay admin tool will update the transaction status.
+
+## Control the payment method
+
+When you create an API key, you choose which payment gateway that key is
+connected with. For example “testing”, “Worldpay”, or “Direct Debit”. You can
+use different API keys to direct paying users to different payment methods.
+
+### Agreement lifecycle
```

Comment:
i think this needs to be an h1? or ## (not sure of the right terminology)

Reply:
this section also feels a little out of place - maybe it needs to go above 'one-off payments using DD' since you'd need to choose what API key to use before embarking on one off or variable DD...

Reply:
Tried moving it up there